src/components/PageHeader/PageHeader.stories.mdx

```diff
@@ -0,0 +1,78 @@
+import { Story, Canvas, Props } from '@storybook/addon-docs/blocks';
+import PageHeader from './PageHeader.jsx';
+
+<Meta title="PageHeader" component={PageHeader} />
+
+# PageHeader
+
+Very large text that can be used to present the title and short additional intro text for a non-article page.
+
+It takes optional children that should be used for relevant non-text components, such as navigation elements.
+
+## Props
+
+<Props of={PageHeader} />
```

Comment:
Use `ArgsTable` for a fun interactive props table. (see Example.stories.mdx). This is new in the PR I just merged this morning, sorry.

Reply:
Oooooh!